ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1038,7 +1038,8 @@ func (r *RayClusterReconciler) buildRedisCleanupJob(instance rayv1.RayCluster) b
 	pod.Spec.Containers = []corev1.Container{pod.Spec.Containers[common.RayContainerIndex]}
 	pod.Spec.Containers[common.RayContainerIndex].Command = []string{"/bin/bash", "-lc", "--"}
 	pod.Spec.Containers[common.RayContainerIndex].Args = []string{
-		"python -c " +
+		"echo \"To get more information about manually delete the storage namespace in Redis and remove the RayCluster's finalizer, please check https://docs.ray.io/en/master/cluster/kubernetes/user-guides/kuberay-gcs-ft.html for more details.\"" +
```

Comment:
Could you check if the key in Redis has actually been deleted? There is no `&&` between the echo command and the Python command. I doubted whether the Python script would be executed or not.

Reply:
Thank you for pointing that out.